apps/ui/src/views/Proposal/Votes.vue

```diff
@@ -209,26 +209,32 @@ watch([sortBy, choiceFilter], () => {
               </div>
             </td>
             <td class="relative">
-              <div
-                v-if="proposal.type !== 'basic'"
-                class="truncate"
-                :title="proposal.choices[vote.choice - 1]"
-              >
-                {{ proposal.choices[vote.choice - 1] }}
+              <div v-if="!!props.proposal.privacy && !props.proposal.completed">
+                <span class="text-skin-heading">Encrypted choice</span>
+                <i-h-lock-closed class="inline-block ml-1" />
```

Comment:
Icon doesn't seem properly aligned 
<img width="243" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/51686767/fa9f7bec-7a5c-4ffd-8f99-c8f495e9de36">

Reply:
Add seems bigger than in the design